### PR TITLE
Spin: fix for stop scroll on swipe

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1168
[Problem] Spin / DateTimePicker stop scroll on long swipe
[Solution]
 - Increasing the number of carousel elements reduces this problem
 - added timeout for lack of dragEnd event and reset drag event detecting

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: src/js/core/widget/core/Spin.js [hunk 1]
```diff
@@ -57,7 +57,7 @@
 			ROLL_DURATION = 600,
 			DELTA_Y = 100,
 			DRAG_STEP_TO_VALUE = 60,
-			NUMBER_OF_CAROUSEL_ITEMS = 11,
+			NUMBER_OF_CAROUSEL_ITEMS = 13,
```
Comment (lslachciak) [inline src/js/core/widget/core/Spin.js:60]: is this change still needed?

File: src/js/core/widget/core/Spin.js [hunk 2]
```diff
@@ -677,13 +678,26 @@
...
+				self._dragEnd({
+					detail: {
+						velocityY: e.velocityY,
+						distance: e.distance,
+						direction: e.direction
+					}
+				});
+				self._dragTimeoutHandler = null;
+				ns.event.gesture.Manager.getInstance().resetDetecting();
+			}, 500);
```
Comment (lslachciak) [inline src/js/core/widget/core/Spin.js]: how about definig this 500 timeout as const together with other const?
see:
ENABLING_DURATION = 300, // [ms]
			ROLL_DURATION = 600,
			ROLL_DURATION = 600,
			DELTA_Y = 100,
			DELTA_Y = 100,
			DRAG_STEP_TO_VALUE = 60,
			DRAG_STEP_TO_VALUE = 60,
			NUMBER_OF_CAROUSEL_ITEMS = 11,
			NUMBER_OF_CAROUSEL_ITEMS = 13,

**Review by lslachciak — COMMENTED** [review]
works very well, just two minor comments

File: src/js/core/widget/core/Spin.js (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/js/core/widget/core/Spin.js:60]: It is not very necessary but extends touching area for drag/swipe.

File: src/js/core/widget/core/Spin.js (hunk 2 above)
Comment (TomaszLukawskiSam) [reply src/js/core/widget/core/Spin.js]: ok. Done

**Review by lslachciak — APPROVED** [review]
